src/site/_includes/components/Glitch.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const {html} = require('common-tags');
+const {escape, stringify} = require('querystring');
+const iframe = require('./IFrame');
+
+/**
+ *
+ * @param {string | { id: string; path?: string; previewSize?: number; style?: string; }} param
+ * @return string
+ */
+module.exports = (param) => {
+  const allow =
+    'camera; clipboard; clipboard-read; clipboard-write; geolocation; encrypted-media; microphone; midi; vr;';
+  let glitchProps = {
+    id: null,
+    path: '',
+    previewSize: null,
```

Comment:
kind of wondering if this should default to 100 as that looks like what the majority of embeds use?